python/ray/tests/test_multi_tenancy.py

```diff
@@ -129,6 +129,19 @@ def get_env(key):
     assert ray.get(get_env.remote("foo2")) == "bar2"
 
 
+def test_override_worker_env(shutdown_only):
+    ray.init(_system_config={"enable_multi_tenancy": True})
```

Comment:
Can you also test if it shows the expected behavior when you have conflicts env vars for job config & worker config?

Reply:
The test case should not be in `test_multi_tenancy.py`. It's not a multi-tenancy feature.

Reply:
I think it is still worth validating how the feature interacts with multi tenancy features right? We'd like to make sure the behavior is consistent when the env var is set by both features.

Reply:
But I do agree the basic feature validation should be written in a different test file. Maybe test_actor_advanced_3.py?